tests/Unit/Utilities/Test_Formaline.cpp

```diff
@@ -0,0 +1,33 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "tests/Unit/TestingFramework.hpp"
+
+#include <string>
+#include <vector>
+
+#include "Utilities/FileSystem.hpp"
+#include "Utilities/Formaline.hpp"
+
+SPECTRE_TEST_CASE("Unit.Utilities.Formaline", "[Unit][Utilities]") {
+  SECTION("check archive size") {
+    const auto archive = formaline::get_archive();
+    // The archive should be at least 1.7MB
```

Comment:
Where does this magic number come from? Is the archive 1.7MB on develop today? Perhaps document the source of this number.